### DISCUSSION TITLE
Issue in uploading file on vercel platform

### DISCUSSION THREAD

**Discussion by harpreet997** [vercel/community#2512] [Help] [answered]
### Summary

I am creating API in Node JS for file upload. So in Local it gets store in uploads folder. But when I deploy the code on vercel and hit the api with deployed url it is giving 504 error code. Unable to find uploads folder.

**Comment by amyegan** [comment, marked as answer]
There isn't a built-in uploads folder on deployed sites. Instead, you can use an external service to store files uploaded via your site. I believe the following guides will help you.
- [How to Upload and Store Files with Vercel](https://vercel.com/guides/how-to-upload-and-store-files-with-vercel)
- [How can I use files in Serverless Functions on Vercel?](https://vercel.com/guides/how-can-i-use-files-in-serverless-functions)
